tests/functional_tests/models/test_gemma2_conversion.py

```diff
@@ -86,18 +86,8 @@ def gemma2_toy_model_path(self, tmp_path_factory):
             print(f"Before save - {name}: {param.dtype}")
             break  # Just check the first parameter
 
-        # Download and save tokenizer from a reference Gemma model
-        # We use the smallest available Gemma model for tokenizer artifacts
-        # First try to load from pre-mounted test data, then fall back to HuggingFace download
-        pre_downloaded_path = "/home/TestData/megatron_bridge/tokenizers/google/gemma-2b"
-        # Try loading from pre-downloaded location first
-        if Path(pre_downloaded_path).exists():
-            print(f"Loading tokenizer from pre-downloaded path: {pre_downloaded_path}")
-            tokenizer = GemmaTokenizer.from_pretrained(pre_downloaded_path)
-        else:
-            # Fall back to downloading from HuggingFace
-            print("Pre-downloaded tokenizer not found, attempting to download from HuggingFace")
-            tokenizer = GemmaTokenizer.from_pretrained("google/gemma-2b")
+        # Download and save tokenizer from a reference Gemma model directly from Hugging Face
+        tokenizer = GemmaTokenizer.from_pretrained("google/gemma-2b")
         tokenizer.save_pretrained(model_dir)
```

Comment:
why is this being removed? these are on the azure mount for CI. is it that not everyone running these tests will have access? in that case, what's the issue with the fallback logic?

Reply:
it should get from HF_HOME folder with Transformer offline, not a custom folder, too much trouble for future.